testbed/coav-sim.sh

```diff
@@ -21,6 +21,11 @@ SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
 # Supported Autopilots: AP_PX4 and AP_APM
 AUTOPILOT=${AUTOPILOT:-"AP_PX4"}
 
+# 0: Log only
+# 1: Log + stdout
+# 2: stdout only
+VERBOSE_LEVEL=${VERBOSE_LEVEL:-0}
```

Comment:
NOTE: Using VERBOSE as variable name conflicts with the make command for jmavsim and adds a lot of unnecessary noise to the output.